aidb/engine/tasti_engine.py

```diff
@@ -95,7 +95,8 @@ def get_score_query_str(self, query: Query, rep_table_name: str) -> (str, List[L
     score_list = []
     # FIXME: for different database, the IN grammar maybe different
     for fp in filering_predicate_score_map:
-      score_list.append(f'IIF({fp}, 1, 0) AS {filering_predicate_score_map[fp]}')
+      # score_list.append(f'IIF({fp}, 1, 0) AS {filering_predicate_score_map[fp]}')
+      score_list.append(f'(CASE WHEN {fp} THEN 1 ELSE 0 END) AS {filering_predicate_score_map[fp]}')
```

Comment:
'IIF' function doesn't exist in either MySQL or PostgreSQL databases.

Reply:
Does this also work on sqlite?

Reply:
Yes